src/main/java/de/retest/recheck/cli/FilterUtil.java

```diff
@@ -27,4 +34,22 @@ public static Filter getExcludeFilterFiles( final List<String> exclude ) {
 		return Stream.concat( excluded, Stream.of( RecheckIgnoreUtil.loadRecheckIgnore() ) ) //
 				.collect( Collectors.collectingAndThen( Collectors.toList(), CompoundFilter::new ) );
 	}
+
+	public static List<String> getInvalidFilters( final List<String> exclude ) {
```

Comment:
As the list contains filters (filter names), the exclude parameter should be named accordingly.